### ISSUE TITLE
HTTP Functions do not work past the first parameter.

### ISSUE THREAD

**Issue by Tivoli** [GoogleCloudPlatform/cloud-functions-emulator#25]
### Description
Cannot read params past initial path for example '/test' works but '/test/123' does not work.

### Steps to reproduce

Create a basic http function and try and navigate to the path with additional params.

Note: this does work on cloud functions with any number or parameters.

Thanks!

**Comment by jmdobry** [comment]
You're using the 0.4.0 version of the Emulator currently available on NPM?

**Comment by Tivoli** [comment]
Yep, I have the latest version installed.

**Comment by jmdobry** [comment]
Can you post some of your function code where you're trying to read the params?

**Comment by jmdobry** [comment]
Or are you trying to curl the function's endpoint with additional paths appending to the url?

**Comment by Tivoli** [comment]
Just opening it in Chrome. It just returns `No function with name test/123` as the copy on the page.

**Comment by Tivoli** [comment]
This is a function that reproduces the error.

```
exports.test = function test(req, res) {
  const url = req.path;
  res.json({url});
}
```

**Comment by jmdobry** [comment]
This should be fixed in 1.0.0-alpha.1, install it with:

    npm install -g @google-cloud/functions-emulator@1.0.0-alpha.1

or

    yarn global add @google-cloud/functions-emulator@1.0.0-alpha.1
